app/models/product.rb

```diff
@@ -10,13 +10,52 @@ class Product < ActiveRecord::Base
   validates :seller_id, presence: true, numericality: { only_integer: true }
   validates :stock, presence: true, numericality: { only_integer: true }
 
+  # Scopes ---------------------------------------------------------------------
+  # scope :active, -> { where("retired: false") }
```

Comment:
because I thought of it for the future but didn't need it yet? we do have a pending card for this & I didn't want to lose the thought.